Frameworks/Foundation/NSOrderedSet.mm

```diff
@@ -16,44 +16,41 @@
 
 #include "Starboard.h"
 #import <StubReturn.h>
+#include <vector>
 
-#include "Foundation/NSOrderedSet.h"
+#import <Foundation/NSOrderedSet.h>
+#import <Foundation/NSException.h>
+
+@implementation NSOrderedSet {
+    std::vector<StrongId<id>> setContainer;
```

Comment:
It is more complicated than that. If you are doing NSMutableOrderedSet (which we should also do), they could share the implementation the iterator becomes important as items can be erased. On that note, you should also consider doing the Mutable version.  Yet another design change would be to do class clustering on this so that the backing store is in an implementation class and not directly on NSOrderedSet.  @bbowman is writing this up with issue #471 as a design. 

---
In reply to: [61966453](https://github.com/Microsoft/WinObjC/pull/472#discussion_r61966453) [](ancestors = 61966453)